#Configuration file for the world_to_base node (w2b) when in the data streaming pane of motive the Up Axis option is set to "Z up"
world_to_base:
  ros__parameters:
      #Orientation of the robot base w.r.t. Motive Frame as unit quaternion
      base_qx: 0.0 
      base_qy: 0.0
      base_qz: 0.0
      base_qw: 1.0
      initial_offset_x: 0.0
      initial_offset_y: 0.0
      initial_offset_z: -0.015 #for new base #-0.019 for old base #unit of measures are in [m]
      base_id: 1 #motive ID of the robot base
      sub_topic: "mocap_rigid_bodies"
      pub_topic: "baseframe_rigid_bodies"